SU2_CFD/src/numerics/NEMO/NEMO_diffusion.cpp

```diff
@@ -336,5 +328,4 @@ CNumerics::ResidualType<> CAvgGradCorrected_NEMO::ComputeResidual(const CConfig
   }
 
   return ResidualType<>(Flux, nullptr, nullptr);
-}
-
+}
```

Comment:
I am not sure whether there are negative consequences of not having a newline at the end of the file, but if GitHub warns about this one might as well just add a newline :)

Reply:
Yes good point, it looks like this is mainly to differentiate a file with/without the newline in the diff, but I guess can also cause issues in some cases if files are concatenated. I'm going to open up a second PR to clean up the NEMO convective schemes, I'll address this fix.